### ISSUE TITLE
hoverSound and pressSound won't work

### ISSUE THREAD

**Issue by Kiiinji** [lumien231/Custom-Main-Menu#394]
Hi, I'm having issues making the hoverSound and pressSound working..
I tried everything : changing the path in the file, old 1.7.10 tutorial on youtube, mp3, ogg, nothing seems to work, it just removes the basic minecraft sound (the button makes no sound)

here's my config :

"buttons":
    {

    	"tanya":
        {
            "text" : "",
            "posX" : -100,
            "posY" : 48,
            "width" : 200,
            "height" : 20,
		    "imageWidth" : 606,
            "imageHeight" : 69,
            "hoverSound" : "modpack:sounds/gunload.ogg",
            "pressSound" : "modpack:sounds/gunfire.ogg",
            "texture" : "modpack:textures/tanya.png",
            //"tooltip" : "Paré au combat !",
            "action" : 
            {
                "type" : "connectToServer",
                "ip" : "26.141.202.39"
            }
        },
``

**Comment by ROMVoid95** [comment]
might require a json file to explicitly state the sound action in custom main menu, take mine for example.
**File Structure**
```bash
└───resources
    └───menu
        ├───sounds.json
        └───sounds
            └───click.ogg
            └───hover.ogg
```

in sounds.json 
```json
{
	"click": {"sounds": ["click"]},
	"hover": {"sounds": ["hover"]}
}
```

then my buttons in the mainmenu configs have it set as shown 
https://hasteb.in/pohavecu.json

**Comment by Kiiinji** [comment]
Alright, still as you can see it's not working :/
https://github.com/lumien231/Custom-Main-Menu/issues/341#issuecomment-565906078

**Comment by ROMVoid95** [comment]
add a `pack.mcmeta` file to the resources folder with this in the file
```json
{"pack":{"pack_format":3,"description":"B.A.S.E External Resources"}}
```
put the sounds folder and json from `modpack` in there own folder in resources as `menu` just the same as the structure i listed

**Comment by Kiiinji** [comment]
I appreciate the help, but I'm sorry to tell you that's it's still not working :( 
I did everything as you told me :
![image](https://user-images.githubusercontent.com/39131872/70881217-243d2d80-1fcc-11ea-9b95-be67294a9b60.png)
![image](https://user-images.githubusercontent.com/39131872/70881229-2a330e80-1fcc-11ea-8f1e-7367209ee4e1.png)
![image](https://user-images.githubusercontent.com/39131872/70881235-2ef7c280-1fcc-11ea-8674-827436298ea4.png)
![image](https://user-images.githubusercontent.com/39131872/70881256-37e89400-1fcc-11ea-9e11-e6c53a71c0e4.png)

**Comment by ROMVoid95** [comment]
is ResourceLoader installed by chance?

**Comment by Kiiinji** [comment]
Of course, my custom main menu is working, custom textures are showing, and my custom background too

**Comment by Kiiinji** [comment]
![image](https://user-images.githubusercontent.com/39131872/70881373-9d3c8500-1fcc-11ea-95a5-f5e013c49f94.png)

**Comment by Kiiinji** [comment]
It removes the button sound, but it won't play my custom sounds :/

**Comment by Kiiinji** [comment]
up :(

**Comment by Kiiinji** [comment]
up

**Comment by ROMVoid95** [comment]
im actually curious about this, i assume you use discord yes?

**Comment by Kiiinji** [comment]
Yes
Kinji#7352

**Comment by Elrol** [comment]
Was a fix ever found?

**Comment by TheMinersMax** [comment]
I'm curious as well, was a fix ever found? running into the same problem myself, though I'm sure I'm missing something small

**Comment by flareblitz21** [comment]
Having the same issue any solutions?

**Comment by RookiesTwo** [comment]
> Having the same issue any solutions?

https://github.com/lumien231/Custom-Main-Menu/issues/205
